cogs/time.py

```diff
@@ -27,12 +27,12 @@ async def time(self, interaction: discord.Interaction):
       rpepoch = -783754059.049 # ..and roleplay epochs respectively, with 91.310625 roleplay seconds passing for
       currentrl = time.time()  # every 1 second in real life. This is what the time command sets out to do.
       currentrp = rpepoch+((currentrl-rlepoch)*91.310625)
-      if datetime.utcfromtimestamp().strftime('%-d %B %Y %H:%M:%S'):
-          readable = datetime.utcfromtimestamp().strftime('%-d %B %Y %H:%M:%S')
+      try:
+          readable = (datetime.utcfromtimestamp(0)+timedelta(seconds=currentrp)).strftime('%d %B %Y %H:%M:%S')
           embed = embed_template("The current date and time is:", readable)
           await interaction.response.send_message(embeds=[embed])
-      except e as Exception:
-          error_embed = error_template(f"Conversion to readable time failed. Fuck.\n{logging.log.error(e)}")
+      except Exception as e:
+          error_embed = error_template(f"Time command failed.\n{logging.log.error(e)}")
```

Comment:
the logging.log stuff will log it out to the console, so seperate it off
also logging.log.exception() on its own will work [not your fault, me being dumb]

Reply:
real